Makefile

```diff
@@ -1,8 +1,11 @@
 EMACS ?= emacs
+EMACS_LOADPATH ?= ~/.emacs.d/elpa/*/
```

Comment:
I believe this should be the default location for people on a Unix-like system.  Perhaps there's a better default value here.  On the other hand, maybe there's a better fix altogether.